echo-events/echo/src/main/java/com/opsmx/plugin/custom/event/EventListenerExtension.java

```diff
@@ -99,4 +99,34 @@ private void ssdEvents(Map<String, Object> eventMap) {
             logger.error("Exception occurred while processing event : {}", e);
         }
     }
+
+    private void ssdStageWithDeploymentManifestEvents(Map<String, Object> eventMap) {
+        boolean pipelineStatus = false;
```

Comment:
change this variable name to isStageStartingEvent

Reply:
done